wegas-app/src/main/webapp/2/src/Editor/Components/ScriptEditors/MergeEditor.tsx

```diff
@@ -0,0 +1,364 @@
+import { css } from 'emotion';
+import * as React from 'react';
+import { themeVar } from '../../../Components/Theme';
+import { Toolbar } from '../../../Components/Toolbar';
+import { IconButton } from '../../../Components/Button/IconButton';
+import DiffEditor, {
+  ExtendedDiffNavigator,
+  DiffEditorLineChanges,
+} from './DiffEditor';
+
+const diffLabel = css({
+  color: themeVar.primaryLighterColor,
+  padding: '5px',
+});
+
+/**
+ * textToArray split a text into an array of lines
+ *
+ * @param text - the text to be splitted
+ */
+const textToArray = (text: string): string[] => {
+  return text.split(/\r|\n/);
```

Comment:
This seems to be the last thing to correct.
```suggestion
  return text.split(/\r?\n/);
```

Reply:
Thx,  I thought I allready did it :s